testblas/common.go

```diff
@@ -14,6 +14,12 @@ func dTolEqual(a, b float64) bool {
 	if math.IsNaN(a) && math.IsNaN(b) {
 		return true
 	}
+	if math.IsInf(a, 1) && math.IsInf(b, 1) {
```

Comment:
Why not just check if a == b for these two cases and return fast for all cases where a is exactly b?